rfcs/0002-ipfs-governance.md

```diff
@@ -0,0 +1,215 @@
+- Feature Name: ipfs-governance
+- Start Date: 2018-02-09
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Establish a basic, adaptable governance structure for IPFS and its surrounding protocols and projects. Design it to minimize bureaucracy, maximize transparency, increase the autonomy of groups across the IPFS ecosystem, and to allow the overall structure of teams/working groups etc. to evolve over time in response to practical needs.
+
+This RFC describes the initial version of the "kernel" of the IPFS + libp2p org(s)
+
+_This RFC is partially motivated by the IPFS Working Groups proposal that @diasdavid submitted in [this PR](https://github.com/ipfs/ipfs/pull/285/files), which predates this repository and the RFC process. Some of the text in this RFC comes from that PR._
+
+# Motivation
+[motivation]: #motivation
+
+## Size and Maturity of the Projects
```

Comment:
If all the text accompanying each of the motivation entries disappeared, and the Motivation section was just a list with 4 bullet points I wouldn't mind.

Reply:
To clarify, there are 3 parts in this document:

* A part to set a context and try to convince readers why it's good to do this
* A part to explain concepts
* A part to set rules

I think the first part (motivation) belongs more to the discussion than to the document itself. The document could have a pointer to it though.